example/scripts/run_test.sh

```diff
@@ -2,22 +2,26 @@
 
 set +e
 
-specName=$(echo "$2" | rev | cut -d'/' -f 1 | cut -d'.' -f 2 | rev)
+specName=$(echo "$3" | rev | cut -d'/' -f 1 | cut -d'.' -f 2 | rev)
 outputPath="${1}/${specName}-running.txt"
+testTimeout=$2
 
 TIMESTAMP=$(date "+%Y-%m-%dT%H:%M:%S")
-echo "$TIMESTAMP ../node_modules/.bin/jasmine $2 STARTED"
-
-../node_modules/.bin/jasmine --no-color "$2" > "$outputPath" 2>&1
+echo "$TIMESTAMP ../node_modules/.bin/jasmine $3 STARTED"
+timeout "$testTimeout" ../node_modules/.bin/jasmine --no-color "$3" > "$outputPath" 2>&1
 result=$?
 
 TIMESTAMP=$(date "+%Y-%m-%dT%H:%M:%S")
 if [ "$result" -eq "0" ]; then
-  echo "$TIMESTAMP ../node_modules/.bin/jasmine $2 PASSED"
+  echo "$TIMESTAMP ../node_modules/.bin/jasmine $3 PASSED"
+  mv "$outputPath" "$1/${specName}-passed.txt"
+elif { [ "$result" -gt "124" ] && [ "$result" -lt "128" ] ;} || [ "$result" -eq "137" ]; then
```

Comment:
What are we saying here?

Reply:
😅  so take a look at the exit codes for timeout https://www.gnu.org/software/coreutils/manual/html_node/timeout-invocation.html#timeout-invocation -- the intent here was to log the failure, except I missed this was moving it to passed.   Remedied: 42bd2e4926

Reply:
Ah ok. That makes more sense 😄